addons/nuxeo-platform-document-routing/nuxeo-routing-api/src/main/java/org/nuxeo/ecm/platform/routing/api/DocumentRoutingService.java

```diff
@@ -588,6 +588,8 @@ void grantPermissionToTaskDelegatedActors(CoreSession session, String permission
      * @param session
      * @return
      * @since 7.2
+     * since 11.1 this method does not scale as it does not paginate results.
```

Comment:
There's missing the `@deprecated` in both javadoc and method, if you don't want it in the backport you can remove this kind of line. Note that we're ok to backport deprecation such as them.

Reply:
We tried to avoid that in order to have warnings on IDE since it is not deprecated for this version.
Does this sound a good approach?

Reply:
So, as per our conversation, we kept the cherry-pick commit.
In order to be clear the difference on 10.10, a separate commit was introduced.